apps/app/src/components/Account/AccountPromotionClaimActions.tsx

```diff
@@ -1,16 +1,21 @@
 import {
+  useSend5792ClaimRewardsTransaction,
+  useSend5792PoolWideClaimRewardsTransaction,
   useSendClaimRewardsTransaction,
   useSendPoolWideClaimRewardsTransaction,
   useToken
 } from '@generationsoftware/hyperstructure-react-hooks'
 import { useAddRecentTransaction, useChainModal, useConnectModal } from '@rainbow-me/rainbowkit'
+import { useMiscSettings } from '@shared/generic-react-hooks'
 import { TokenAmount, TransactionButton } from '@shared/react-components'
 import { getSecondsSinceEpoch, lower } from '@shared/utilities'
 import classNames from 'classnames'
 import { useTranslations } from 'next-intl'
 import { useMemo } from 'react'
 import { Address, formatUnits } from 'viem'
 import { useAccount } from 'wagmi'
+import { useCapabilities } from 'wagmi/experimental'
```

Comment:
Wagmi/viem now have stable release for EIP-5792 beginning with viem@2.28 and wagmi@2.15 and later. 

```ts
import { useCapabilities } from 'wagmi'
```
https://wagmi.sh/react/api/hooks/useCapabilities

Similarly for the other hooks: useSendCalls, etc..

Reply:
Updated in [7d38773](https://github.com/GenerationSoftware/pooltogether-client-monorepo/pull/106/commits/7d38773985ff18ca25d63f721b2a990a7b34d933)